lib/src/main/java/uk/gov/di/ipv/cri/address/library/service/AddressService.java

```diff
@@ -71,4 +88,104 @@ private ObjectReader getAddressReader() {
         }
         return this.addressReader;
     }
+
+    public void setAddressValidity(List<CanonicalAddress> addresses)
+            throws AddressProcessingException {
```

Comment:
Add a comment with a reference to the docs we looked at in our call last week

Reply:
Link is added